_1327/emails/templates/emails_search.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base_without_sidebar.html" %}
+{% load i18n %}
+{% load static %}
+{% load compile_static %}
+{% load mptt_tags %}
+{% load bootstrap3 %}
+
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'less/emails.less'|compile %}" />
+{% endblock %}
+
+{% block title %}{% trans 'Search emails' %}{% endblock title %}
+
+{% block content %}
+	{{ block.super }}
+
+	<form method="post" action="#results" class="form">
+		{% bootstrap_form search_form %}
+		{% csrf_token %}
+		{% buttons %}
+			<button type="submit" class="btn btn-primary">{% trans 'Search' %}</button>
+		{% endbuttons %}
+	</form>
+
+	<h2 id="results">{% trans 'Search results' %}</h2>
+	{% if emails|length == 0 %}
+		<div class="alert alert-warning">{% trans 'No emails found for the given search query.' %}</div>
+	{% else %}
+		<ul class="email-tree">
+		{% for email in emails %}
+			<li>
+				<a href="{% url 'emails:view' email.id %}">{{ email.subject_nonempty }}</a>
```

Comment:
can you include the sender in this line like everywhere else? it would also be nice if you could link to a view page that highlights found search terms in the text but it's not really required for the first version of this implementation :)